orm/models.py

```diff
@@ -156,13 +156,22 @@ def select_related(self, related):
             select_related=related,
         )
 
+    async def exists(self) -> bool:
+        expr = self.build_select_expression()
+        expr = sqlalchemy.exists(expr).select()
+        exists, = await self.database.fetch_one(expr)
+        return exists
```

Comment:
Do we need to cast this to a bool or will it already be one?

Reply:
It’s already returned as a boolean value — so yes for specifying that in the tests. 👍